ObjectMapper/Core/Mapper.swift

```diff
@@ -334,18 +334,18 @@ extension Mapper {
 	public func toJSONString(_ object: N, prettyPrint: Bool = false) -> String? {
 		let JSONDict = toJSON(object)
 		
-        return Mapper.toJSONString(JSONDict, prettyPrint: prettyPrint)
+        return Mapper.toJSONString(JSONDict as Any, prettyPrint: prettyPrint)
```

Comment:
You can avoid all of these casts if we change the declaration: https://github.com/Hearst-DD/ObjectMapper/compare/swift-3...joshpc:swift-3?expand=1#diff-a9498ca83904e6eb2c05ea55fe3c5a1dR348